src/python/bindings.cpp

```diff
@@ -40,4 +45,8 @@ PYBIND11_MODULE(pyqusat, m) {
         "Check the equivalence of two clifford circuits for the given inputs."
         "If no inputs are given, the all zero state is used as input.",
         "circ1"_a, "circ2"_a, "inputs"_a = std::vector<std::string>());
+
+  m.def("generate_dimacs", &printDIMACS,
+        "Output the DIMACS CNF representation from Z3 of the given circuit.",
+        "circ1"_a);
```

Comment:
```suggestion
        "circ"_a);
```
Given how there is only a single circuit in this method this seems like a more suitable name.